### PR TITLE
slimerjs 0.9.5

### PR DESCRIPTION
**Pull request by bfontaine**

### CODE REVIEW

**Comment by bfontaine** [comment]
I fixed the test because SlimerJS needs either Firefox or to be built with `xulrunner` (see @mistydemeo’s comment on #36149).

File: Library/Formula/slimerjs.rb [hunk 1]
```diff
@@ -24,27 +22,34 @@ def install
...
       #{libexec}/application.ini
     EOS
   end
 
   test do
-    system "#{bin}/slimerjs", "-v"
-    curl "-O", "https://raw.githubusercontent.com/laurentj/slimerjs/ec1e53a/examples/phantomjs/loadspeed.js"
-    system "#{bin}/slimerjs", "loadspeed.js", "https://www.google.com"
+    firefox = "/Applications/Firefox.app/Contents/MacOS/firefox"
+    has_firefox = File.executable? firefox
```
Comment (mikemcquaid) [inline Library/Formula/slimerjs.rb]: We can assume `firefox` isn't around as this should be optimised for running on the bot which won't have it.

**Comment by mikemcquaid** [comment]
`xulrunner` won't download it seems: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/20612/version=yosemite/testReport/junit/brew-test-bot/yosemite/fetch_gnu_tar_nspr_nss_pkg_config_python_python_xulrunner_yasm/

**Comment by bfontaine** [comment]
@mikemcquaid Right, I fixed its URL in #36359.

**Comment by bfontaine** [comment]
@mikemcquaid Is it normal the bot tries ([and fails](http://bot.brew.sh/job/Homebrew%20Pull%20Requests/20665/version=yosemite/console)) to build on Yosemite when `xulrunner`, a `slimerjs` dependency, requires OS X <= Mavericks?

```
==> brew install --only-dependencies --verbose --build-bottle slimerjs	 FAILED
xulrunner: OS X Mavericks or older is required.
```

**Comment by mikemcquaid** [comment]
It's probably normal but not planned. The requirements exclusion stuff is pretty broken. Pulling, thanks!

**Pull request closed without merging**